### DISCUSSION TITLE
What does ./... (dot slash dot dot dot) mean in the Kubernetes tutorial?

### DISCUSSION THREAD

**Discussion by staticaland** [cue-lang/cue#1868] [Q&A] [answered]
In [the Kubernetes tutorial](https://github.com/cue-lang/cue/tree/v0.4.3/doc/tutorial/kubernetes) there is this kind of syntax with three dots (or periods):

```sh
cue import ./... -p kube
```

What does it mean exactly? I found it hard to find documentation about it.

**Comment by mvdan** [comment, marked as answer]
See `cue help inputs` :) It is referenced from both `cue -h` and `cue help`.

**Reply by mvdan** [reply]
But to answer your question more directly, `./...` is a shorthand for "all packages under the current directory", since the `./` prefix means the current directory, and `...` means "match anything".
